stix_shifter_modules/qradar/entry_point.py

```diff
@@ -11,4 +11,4 @@ def __init__(self, connection={}, configuration={}, options={}):
         else:
             self.setup_translation_simple(dialect_default='flows')
             dialect = 'aql'
-            self.add_dialect(dialect,  AqlQueryTranslator(options, dialect, None), default_include=False, default=True)
+            self.add_dialect(dialect,  AqlQueryTranslator(options, dialect, None), default_include=True)
```

Comment:
wondering what is the difference between default and default_include and how does it work after the entry point?

Reply:
one dialect per language has to have 'default=True' to be used for 'parse' calls since for calls like:
`python main.py translate qradar parse '{}' "[url:value = 'https://google.com/']" '{"language":"stix"}'`
dialect is not specified in cli. parsing will be processed with dialect='flows' since it is the default one for 'stix' language.